src/03-dice-roller/DiceRoller/DiceRoller/Dice.cs

```diff
@@ -8,26 +8,22 @@ namespace DiceRoller
 {
     public class Dice
     {
-        public static Dice D6 => new Dice(6, "d6");
-
-        public static Dice D8 => new Dice(6, "d8");
-
+        //class variables
         public int Sides { get; }
```

Comment:
This probably shouldn't be public because no-one externally needs to know about the implementation details of a dice.